sdk/python/feast/on_demand_feature_view.py

```diff
@@ -62,24 +62,24 @@ class OnDemandFeatureView(BaseFeatureView):
     """
 
     name: str
-    features: List[Field]
```

Comment:
these types of change will break environment using py39, only support from py310